docs/deploying-to-azure.md

```diff
@@ -202,10 +209,434 @@ Get the connection string with the following command so you can connect to it:
 ```shell
 az postgres flexible-server show-connection-string \
   --database-name $POSTGRES_DB \
-  --query connectionStrings.jdbc
+  --query connectionStrings.jdbc \
+  --out tsv
+```
+
+### Create the managed MongoDB Database
+
+We need to create a MongoDB so the Fight microservice can store data.
+Create a database in the region where it's available:
+
+```shell
+az cosmosdb create \
+  --resource-group $RESOURCE_GROUP \
+  --locations regionName="$LOCATION" failoverPriority=0 \
+  --name $MONGO_DB \
+  --kind MongoDB \
+  --server-version 4.0
+```
+
+Create the Fight collection:
+
+````shell
+az cosmosdb mongodb database create \
+  --resource-group $RESOURCE_GROUP \
+  --account-name $MONGO_DB \
+  --name $FIGHTS_DB_SCHEMA
+````
+
+### Create the Managed Kafka
+
+The Fight microservice communicates with the Statistics microservice through Kafka.
+We need to create an Azure event hub for that.
+
+```shell
+az eventhubs namespace create \
+  --resource-group $RESOURCE_GROUP \
+  --location $LOCATION \
+  --name $KAFKA_NAMESPACE
+```
+
+```shell
+az eventhubs eventhub create \
+  --resource-group $RESOURCE_GROUP \
+  --name $KAFKA_TOPIC \
+  --namespace-name $KAFKA_NAMESPACE
+```
+
+```shell
+APICURIO_URL=$(az containerapp create \
+  --resource-group $RESOURCE_GROUP \
+  --image apicurio/apicurio-registry-mem:2.1.5.Final \
```

Comment:
Version should be `2.2.0.Final` now that `main` uses Quarkus `2.8.0.Final`.

Reply:
Done